src/client/containers/Home/Home.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable no-unused-vars */
 import React from 'react';
+import { BrowserRouter as Router, Redirect } from 'react-router-dom';
 
-export const Home = () => <section className="home">Final project!</section>;
+export const Home = () => <Redirect to="/welcome" />;
```

Comment:
better make it /profile
Then it won't jump to welcome page if you are already logged in